app/views/metrics/index.html.erb

```diff
@@ -262,6 +266,21 @@
       </div>
     </div>
   </div>
-  <br />
-  <button class="ui disabled submit button" id="save_configs_btn">SAVE</button>
+  <h4> Access Control </h4>
+  <h6>
+    Providing Course Assistants (CAs) with access will allow them to use the watchlist, but they will not be able to modify the metrics or the configurations.
+  </h6>
+  <div>
+    <div class="ui checkbox <%=@allow_ca ? 'checked' : ''%> " id="allow_ca">
```

Comment:
Nit: remove extra space after `%>`

Also, this feels slightly hacky. It is weird that the `metric_config_endpoints` are called `update_watchlist_configuration` and `get_watchlist_category_blocklist` -- the `get` method should really mirror the update method and be renamed `get_watchlist_configuration`. This would be in line with the `metrics_endpoints` of `update_current_metrics` and `get_current_metrics`.

Then, the `get` method should include the value of `allow_ca`, and the checkbox value should be programatically set.

However, this is a bit of refactoring, and it's fine to leave this for another PR.

Reply:
Makes sense. I refactored based on your suggestion